flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/source/StoreTableSource.java

```diff
@@ -0,0 +1,215 @@
+        this.streaming = streaming;
+        this.logSourceProvider = logSourceProvider;
+    }
+
+    @Override
+    public ChangelogMode getChangelogMode() {
+        return streaming
+                ? tableStore.valueCountMode() ? ChangelogMode.all() : ChangelogMode.upsert()
+                : ChangelogMode.insertOnly();
+    }
+
+    @Override
+    public ScanRuntimeProvider getScanRuntimeProvider(ScanContext scanContext) {
+        TableStore.SourceBuilder builder =
+                tableStore
+                        .sourceBuilder()
+                        .withContinuousMode(streaming)
+                        .withHybridMode(streaming && logSourceProvider != null)
+                        .withLogSourceProvider(logSourceProvider)
+                        .withProjection(projectFields)
+                        .withPartitionPredicate(
+                                partitionFilters != null
+                                        ? partitionFilters.stream()
+                                                .map(
+                                                        filter ->
+                                                                filter.accept(
+                                                                        PredicateConverter
+                                                                                .CONVERTER))
+                                                .reduce(And::new)
+                                                .orElse(null)
+                                        : null)
+                        .withFieldPredicate(
+                                fieldFilters != null
+                                        ? fieldFilters.stream()
+                                                .map(
+                                                        filter ->
+                                                                filter.accept(
+                                                                        PredicateConverter
+                                                                                .CONVERTER))
+                                                .reduce(And::new)
+                                                .orElse(null)
+                                        : null);
+        return SourceProvider.of(builder.build());
+    }
+
+    @Override
+    public DynamicTableSource copy() {
+        StoreTableSource copied = new StoreTableSource(tableStore, streaming, logSourceProvider);
+        copied.partitionFilters = partitionFilters;
+        copied.fieldFilters = fieldFilters;
+        return copied;
+    }
+
+    @Override
+    public String asSummaryString() {
+        return "StoreTableSource";
+    }
+
+    @Override
+    public Result applyFilters(List<ResolvedExpression> filters) {
+        if (tableStore.partitioned()) {
+            classifyFilters(filters);
+        } else {
+            fieldFilters = filters;
+        }
+        return Result.of(
```

Comment:
The filter is a best effort, so just return:
`return Result.of(new ArrayList<>(filters), new ArrayList<>(filters));`
Accept all, and remain all. Planner will do filter again.